data.json

```diff
@@ -126,5 +126,13 @@
     "reference": "https://developers.whatismybrowser.com/api/swag/",
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "tags": ["stickers"]
+  },
+  {
+    "name": "Scalyr",
+    "difficulty": "hard",
+    "description": "<a href='https://www.scalyr.com/signup'>Sign up</a> for a 30days Scalyr trial and get a free t-shirt!",
+    "reference": "https://www.scalyr.com/signup",
+    "image": "https://www.scalyr.com/assets/images/scalyr-logo-color-reverse.png",
```

Comment:
Here's the possible graphics of the t-shirt: https://twitter.com/scalyr/status/1040286617733345280

Maybe we can take only one of the two for illustrating?

Reply:
Here a cut

![dm_xkkxu8aayga0 1](https://user-images.githubusercontent.com/8191198/47256123-ce79fd80-d47a-11e8-92b3-e470f9bc9aa2.jpg)